tests/test_secretsmanager/test_secretsmanager.py

```diff
@@ -752,6 +752,7 @@ def test_put_secret_value_maintains_description_and_tags():
         {"Key": "Mykey", "Value": "Myvalue"},
     ]
     assert secret_details["Description"] == "desc"
+    assert secret_details["VersionIdsToStages"] is not None
```

Comment:
Could you assert that it matches the expected dict, whatever that is?

There's quite a bit of logic around versions/stages (which I'm not sure is completely implemented..), so it would be good to be specific in what moto currently supports.